docs/uptime-guide/deployment-arch.asciidoc

```diff
@@ -0,0 +1,13 @@
+[role="xpack"]
+[[uptime-deployment-arch]]
+== Deployment Architecture
+
+There are infinite ways to deploy Uptime and Heartbeat, some of which work better than others. A guiding principle in these recommendations is that an outage that takes down the service being monitored should not also take down Heartbeat. You will want Heartbeat to be functioning even when your site is not, so the guidelines that follow attempt to minimize this possibility.
+
+Heartbeat is generally run as a centralized service within a datacenter. While it is possible to run it as a separate "sidecar" process paired with each process/container we recommend against it. Running Heartbeat centrally ensures you will still be able to see monitoring data in the event of an overloaded, disconnected, or otherwise malfunctioning server. 
```

Comment:
Consistency, and rewording slightly to clearly state up-front that we do **not** recommend something. 

```suggestion
Heartbeat is generally run as a centralized service within a data center. While it is possible to run it as a separate "sidecar" process paired with each process or container, we do not recommend this. Running Heartbeat centrally ensures you will still be able to see monitoring data in the event of an overloaded, disconnected, or otherwise malfunctioning server. 
```